pkg/internal/testing/process/arguments.go

```diff
@@ -0,0 +1,313 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package process
+
+import (
+	"bytes"
+	"html/template"
```

Comment:
I assume, this is should actually use `text/template`, but uses `html/template` by accident (was already the case before this PR).
This might result in some undesired encoding of special characters command line arguments. I was bitten by this one time, but I actually don't remember what character was encoded.
So just leaving this side-note here, in case you wanna change it :)

Reply:
keeping it the same for now, since we'll be deprecating this part eventually anway.